### PR TITLE
Docusaurus kits

### PR DESCRIPTION
**Pull request by bcronin90**
Add documentation kit for use in Docusaurus.

There is one pain point right now: Significant parts of the docusaurus documentation are similar or identical to existing documentation. If anyone has a smart idea how to de-duplicate this, please tell me.

<sub>Brendan Cronin <brendan.cronin@mercedes-benz.com>, Mercedes-Benz Tech Innovation GmbH, [legal info/Impressum](https://github.com/mercedes-benz/foss/blob/master/PROVIDER_INFORMATION.md) </sub>

### CODE REVIEW

File: docs/kit/operation-view/page01_business_prerequisites.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Business Prerequisites
+
+## BPN
+
```
Comment (bcronin90) [inline docs/kit/operation-view/page01_business_prerequisites.md:4]: @stephanbcbauer This needs to be added

File: docs/kit/adoption-view/page_adoption-view.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+---
+
+The ConnectorKit provides a connector framework, based on the Eclipse Dataspace Connector (EDC) for sovereign, cross-enterprise data exchange.
+
+
+![EDC Overview](../adoption-view/images/edc_overview.png)
+
+Trust, interoperability and data sovereignty, these are the objectives and values for secure and sustainable peer-to-peer data exchange between organizations and companies. The claim is data sovereignty: Whoever makes data available retains control and decides individually who is involved in the data exchange, how, when, where and under what conditions.
+
+A corresponding concept was developed in the context of Gaia -X and the International Data Space Association (IDSA). The essential software component is the connector.
```
Comment (bcronin90) [inline docs/kit/adoption-view/page_adoption-view.md:15]:
```suggestion
A corresponding concept was developed in the context of Gaia-X and the International Data Space Association (IDSA). The essential software component is the connector.
```

File: docs/kit/adoption-view/page_adoption-view.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+The main difference between the EDC and the previous connectors of the IDSA is the separation of the communication into a channel for the metadata and one for the actual data exchange. The channel for the data supports various transmission protocols via so-called data plane extensions. The metadata is transmitted directly via the EDC interface, while the actual data exchange then takes place via the appropriate channel extension. In this way, a highly scalable data exchange is made possible.
+
+![EDC Architecture](../adoption-view/images/edc_architecture.png)
+
+The architecture of the EDC combines various services that are necessary for the above principles:
+
+- An interface to the Identity Provider service, currently IDSA's Dynamic Attribute Provisioning System (DAPS). This central service provides the identity and the corresponding authentication of the participants in the data exchange. (There is no authorization at this point). Decentralized solutions will also be supported in the future.
```
Comment (bcronin90) [inline docs/kit/adoption-view/page_adoption-view.md:38]: In the context of EDC, I would call the contract agreement and its policies an authorization. Though the enforcement of these is kind of intransparent. Maybe the others can comment on this?

File: docs/kit/adoption-view/page_domain_model.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
+---
+id: Domain Model
+title: Domain Model
+description: 'Connector Kit'
+sidebar_position: 2
+---
+
+:::info TBD
```
Comment (bcronin90) [inline docs/kit/adoption-view/page_domain_model.md:8]: TBD?

File: docs/kit/adoption-view/page_domain_model.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+> The shown picture illustrates only a generic view of the Domain Model and is not intended to show all aspects of the project.
+
+## Asset
+
+An asset represents data (databases, files, cache information, etc.) which should be published and shared between
+organizations. For each asset, a [`DataAddress`](#data-address) needs to be resolvable.
+
+## Data address
+
+A data address is a pointer into the physical storage location where an asset will be stored
```
Comment (bcronin90) [inline docs/kit/adoption-view/page_domain_model.md:22]: This feels like the start of a paragraph. Is there more?

File: docs/kit/adoption-view/page_adoption-view.md (hunk 3 above)
Comment (jimmarino) [reply docs/kit/adoption-view/page_adoption-view.md:38]: In EDC there is **access** policy (not advertised) and **usage policy** (advertised), which is codified by a `ContractDefinition`. The union of those determines if a client can acquire an asset. HTH.

File: docs/kit/development-view/openAPI/management-api/cancel-negotiation.api.mdx [hunk 6]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (DominikPinsel) [inline docs/kit/development-view/openAPI/management-api/cancel-negotiation.api.mdx:1]: Maybe we can document how these files are generated. There is a release documentation started at /docs/release. The idea is to have 
- a checklist of things to do, before releasing a software
- a how to release the software

So this should be the right place

**Review by DominikPinsel — APPROVED** [review]
Great PR :) Will merge after the open comments are resolved!

File: docs/kit/adoption-view/page_domain_model.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+* ### Contract agreement
+
+  A contract agreement represents the agreed-upon terms of access and usage of an asset's data between two data space
+  participants, including a start and an end date and further relevant information.
+
+## Policy
+
+Contract policies represent permitted and prohibited actions over a certain asset. These actions can be limited further
+by constraints (temporal or spatial) and duties ("e.g. deletion of the data after 30 days"). Further information is
+provided in a separate [section](usage-control/policies.md).
```
Comment (bcronin90) [inline docs/kit/adoption-view/page_domain_model.md:61]: broken link

File: docs/kit/operation-view/page01_business_prerequisites.md (hunk 1 above)
Comment (stephanbcbauer) [reply docs/kit/operation-view/page01_business_prerequisites.md:4]: @bcronin90 i had a discussion with Daniel Miehle. This is something which should be on another layer. So not in the connectorKit. This file can be deleted

File: docs/kit/adoption-view/page_domain_model.md (hunk 4 above)
Comment (stephanbcbauer) [reply docs/kit/adoption-view/page_domain_model.md:8]: @bcronin90 lets say this is a debugging comment :) We can delete this tbd

File: docs/kit/adoption-view/page_domain_model.md (hunk 5 above)
Comment (stephanbcbauer) [reply docs/kit/adoption-view/page_domain_model.md:22]: @bcronin90 not sure how this is meant? Lets have a call

File: docs/kit/adoption-view/page_domain_model.md (hunk 7 above)
Comment (stephanbcbauer) [reply docs/kit/adoption-view/page_domain_model.md:61]: @bcronin90 the information itself is taken from upstream edc. So the link also references to this page. The information itself is valid, but I think we can delete the link?

File: docs/kit/development-view/openAPI/management-api/cancel-negotiation.api.mdx (hunk 6 above)
Comment (stephanbcbauer) [reply docs/kit/development-view/openAPI/management-api/cancel-negotiation.api.mdx:1]: @DominikPinsel yes and no :) the problem here…. the files are not generated within our repository. They are created via docusaurus plug-ins.?

https://www.npmjs.com/package/docusaurus-plugin-openapi-docs

File: docs/kit/operation-view/page01_business_prerequisites.md (hunk 1 above)
Comment (bcronin90) [reply docs/kit/operation-view/page01_business_prerequisites.md:4]: I think it'd be sensible to at least link to this information. If I wanted to set up an EDC right now, I frankly wouldn't know where to get a BPN.

File: docs/kit/development-view/openAPI/management-api/cancel-negotiation.api.mdx (hunk 6 above)
Comment (bcronin90) [reply docs/kit/development-view/openAPI/management-api/cancel-negotiation.api.mdx:1]: I'll document it based on that.

**Comment by bcronin90** [comment]
Now that this repo has been declared dead effective immediately, what should happen to PRs like this? @stefan-ettl

**Comment by ndr-brt** [comment]
> Now that this repo has been declared dead effective immediately, what should happen to PRs like this? @stefan-ettl

You can re-create the PR on https://github.com/catenax-ng/tx-tractusx-edc
I went with `git format-patch` and `git apply` and was spretty smooth

**Comment by bcronin90** [comment]
> > Now that this repo has been declared dead effective immediately, what should happen to PRs like this? @stefan-ettl
> 
> You can re-create the PR on https://github.com/catenax-ng/tx-tractusx-edc I went with `git format-patch` and `git apply` and it was pretty smooth

Should I be able to push a branch to tractusx-edc? Because I get a 403. Or do I need to rebase a branch from here onto their main and then PR from that?

**Comment by florianrusch-zf** [comment]
I closed this PR, because we already moved to https://github.com/eclipse-tractusx/tractusx-edc/

**Pull request closed without merging**